### DISCUSSION TITLE
See normalized cache

### DISCUSSION THREAD

**Discussion by matthizou** [urql-graphql/urql#2955] [Q&A] [answered]
Hello 👋 
I am using `graphcache` along with the urql Dev Tools chrome extension.  
There is something I really can't figure out, though, is: "How do I see the normalized cache"?

The extension shows me the data retrieved by my queries, whether the cache has been hit, etc. 
But I can't find how to see the cache itself, am I missing something obvious? 😅 

I am looking for something like the following (according to the docs, that's should be the shape of my basic todo app): 

![CleanShot 2023-02-13 at 23 15 26](https://user-images.githubusercontent.com/6507291/218586838-d8c685cb-043a-43b3-bcab-0cf9369a2a02.png)

Thanks in advance

**Comment by JoviDeCroock** [comment, marked as answer]
Currently we have no way to view the contents of the normalized cache, the devtools do an attempt at visualizing what it has seen already. This is something I would love in the future but it's currently not supported, we are in process of updating the devtools extension as it was pretty neglected in terms of dependencies 😅

**Reply by matthizou** [reply]
Thanks for the quick answer! Good luck with the update
